### ISSUE TITLE
Error: Found comma but its following sibling is not a line: undefined

### ISSUE THREAD

**Issue by u3u** [electrovir/prettier-plugin-newline-arrays#42]
```ts
const arr = [
  1,
  // 2,
  3,
];

```

When commenting on the item in the middle of the array, an error will occur during formatting:

```sh
Error: Found comma but its following sibling is not a line: undefined
    at prettier-plugin-multiline-arrays/dist/printer/insert-new-lines.js:234:39
```

**Comment by electrovir** [comment]
Thanks for the report! I'll take a look.

**Comment by Alex-Sokolov** [comment]
Same too :) looks like plugin will fail when there is comment inside array
<img width="727" alt="SCR-20240902-qjdh" src="https://github.com/user-attachments/assets/ec476135-4fdf-459a-b129-66c9eca36f7d">
<img width="1086" alt="SCR-20240902-qjrb" src="https://github.com/user-attachments/assets/33a29d06-62a3-4d72-b861-033b16f5ec02">

Sometimes errors message a little different:
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/b95abfb0-bc9f-4f6b-84de-1feee3c47b11">

"prettier": "3.3.3",
"prettier-plugin-multiline-arrays": "3.0.6",

**Comment by electrovir** [comment]
Fixed for the next release.

**Comment by electrovir** [comment]
released in [v4.0.0](https://www.npmjs.com/package/prettier-plugin-multiline-arrays/v/4.0.0)
